apps/ledger-live-mobile/src/components/Carousel/Slide.tsx

```diff
@@ -1,40 +1,17 @@
 import React, { useCallback } from "react";
-import {
-  Linking,
-  Image,
-  ImageSourcePropType,
-  StyleProp,
-  ImageStyle,
-} from "react-native";
+import { Linking, Image } from "react-native";
 import { Flex, Text, Link as TextLink, Icons } from "@ledgerhq/native-ui";
 import { useTranslation } from "react-i18next";
 import styled from "styled-components/native";
-import { NavigationProp, useNavigation } from "@react-navigation/native";
-
-// eslint-disable-next-line import/no-cycle
+import { useNavigation } from "@react-navigation/native";
 import Touchable from "../Touchable";
-// eslint-disable-next-line import/no-cycle
 import { track } from "../../analytics";
+import { SlideProps } from "./types";
```

Comment:
all changes in the `Carousel` directory are unrelated to the general goal of this PR but there was a require cycle here that was itching me 🧹